src/main/kotlin/com/procurify/flagcounter/EnvironmentTeamParser.kt

```diff
@@ -0,0 +1,50 @@
+package com.procurify.flagcounter
+
+import com.fasterxml.jackson.databind.DeserializationFeature
+import com.fasterxml.jackson.databind.ObjectMapper
+import com.fasterxml.jackson.module.kotlin.KotlinModule
+
+/**
+ * Object for parsing JSON in Env var into a map of teams to webhook URLs
+ * TODO Everything is currently configured in env vars but this could be removed on moving to something like SSM
+ */
+object EnvironmentTeamParser {
+
+    /**
+     * Parse an environment variable stored as a JSON object such as
+     * {
+     *   "teamsList": [
+     *     {
+     *       "email":"a@test.com",
+     *       "url": "google.com"
+     *     },
+     *     {
+     *       "email":"b@test.com",
+     *       "url":"bing.com"
+     *     }
+     *   ]
+     * }
+     * into a Map<TeamEmail, SlackWebhookUrl>
+     */
+    fun parseJsonIntoTeamsMap(jsonString: String): Map<String, String> =
```

Comment:
not a bad idea. 
I was thinking that if we were to ever support other systems, this would only need to be a key in whatever flag system we were using